lib/note-toolbar/index.tsx

```diff
@@ -81,6 +84,13 @@ export class NoteToolbar extends Component<Props> {
         </div>
         {isOffline && <div className="offline-badge">OFFLINE</div>}
         <div className="note-toolbar__column-right">
+          <div className="note-toolbar__button">
+            <IconButton
+              icon={<ChecklistIcon />}
+              onClick={() => window.dispatchEvent(new Event('toggleChecklist'))}
+              title={`Insert Checklist • ${CmdOrCtrl}+Shift+C`}
```

Comment:
I believe we already have a Redux action for adding a checklist. If you use `INSERT_TASK` "the system" should handle the machinery of communicating with the editor for you and then we can avoid the events and listeners

Reply:
If only, if only

Reply:
(Sandy asked me how to do this last night and I was like "oh that seems straightforward, surely there's a simple way to--- ....oh.)

Reply:
was there a problem with the Redux action?

Reply:
It looks like[ Kats message](https://github.com/Automattic/simplenote-electron/pull/2603#discussion_r565535022) could have been edited after the last reply, wanted to comment again pointing to it.

Reply:
Apparently I should not use GitHub comments as a chat client

Reply:
modified Zawinski's law

> Every program attempts to expand until it can ~read mail~ chat.